databricks/koalas/tests/test_series.py

```diff
@@ -1824,6 +1824,35 @@ def test_squeeze(self):
         kser = ks.from_pandas(pser)
         self.assert_eq(kser.squeeze(), pser.squeeze())
 
+    def test_swaplevel(self):
+        # MultiIndex with two levels
+        arrays = [[1, 1, 2, 2], ["red", "blue", "red", "blue"]]
+        pidx = pd.MultiIndex.from_arrays(arrays, names=("number", "color"))
+        pser = pd.Series(["a", "b", "c", "d"], index=pidx)
+        kser = ks.from_pandas(pser)
+        self.assert_eq(pser.swaplevel(), kser.swaplevel())
+        self.assert_eq(pser.swaplevel(0, 1), kser.swaplevel(0, 1))
+        self.assert_eq(pser.swaplevel("number", "color"), kser.swaplevel("number", "color"))
```

Comment:
Just curious: what if we specify the same value for both arguments? `ser.swaplevel(1, 1)`.

Reply:
`IndexError: Too many levels: Index of the series has only 1 levels, -2 is not a valid level number`.

That's a good test case! Let me add it.

Reply:
`self.assert_eq(kser, kser.swaplevel(1, 1))`

The series wouldn't change. Added test cases for `ser.swaplevel(1, 1)`.